pkgs/development/python-modules/pytorch-lightning/default.nix

```diff
@@ -35,6 +41,10 @@ buildPythonPackage rec {
   # models, which doesn't work in the sandbox.
   doCheck = false;
 
+  pipInstallFlags = [
+    "--no-deps" # pipInstallPhase fails looking for setuptools
```

Comment:
This would ignore all other dependencies. We don't want to do that. Please try patching it or repacking the wheel worst case.

Reply:
- There doesn't seem to be anything to patch in this package (only `pyproject.toml` mentions setuptools as a requirement, but removing that doesn't help with the error; and tbh, this sort of absurd errors is why I'd rather not rely on pip for checking dependencies)
- Repacking the wheel sounds like too big of a hammer to implement at a single sdist package level.
- My concern is just to fix pytorch-lightning which is broken right now

Reply:
That is not witchcraft, we are just not on the latest version and github search only searches master https://github.com/PyTorchLightning/pytorch-lightning/blob/1.5.10/requirements.txt#L14

Reply:
Yes, you were right, I looked at the sources of the wrong revision